dsls/src/main/scala/ch/epfl/directembedding/test/example/package.scala

```diff
@@ -121,24 +121,30 @@ package object example {
     def __valDef(v: Int): Int = ???
   }
 
+  object Config extends ExampleConfig
+
   object implementations {
     def liftRep[T](c: Context)(block: c.Expr[T]): c.Expr[T] = {
       import c.universe._
       val config = "ch.epfl.directembedding.test.example"
+      val debug = block.tree.find(_.equalsStructure(
+        q"""
+             "debug"
```

Comment:
This was in my early macro days :) I don't use this anymore, now I have two macros where one explicitly passes true for debug.